### DISCUSSION TITLE
1st cutscene in the game freezes - is there any workaround?

### DISCUSSION THREAD

**Discussion by erdies** [JACoders/OpenJK#1108] [Q&A]
Hi all,
thank you for this linux port. It looks great. I even love these older games. Unfortunatly I have an issue:

I installed compile this code on gentoo amd64 and used the game files from the original windows CD. The game runs fine but the first cutscene in the tempel just hangs when the camera looks inside from the top to the actors.
It would be nice if somebody knows a workaround for this. Is there any way to skip a cutscene? Thanks.

**Comment by Calinou** [comment]
See https://github.com/JACoders/OpenJK/issues/660 and https://github.com/JACoders/OpenJK/issues/193.
